modules/gapi/include/opencv2/gapi/ocl/goclkernel.hpp

```diff
@@ -68,9 +68,14 @@ class GAPI_EXPORTS GOCLContext
     {
         return outVecRef(output).wref<T>();
     }
+    template<typename T> T& outOpR(int output) // FIXME: the same issue
```

Comment:
GArray was here, so I think we can handle GOpaque too. But not sure if was needed